nf_requirements/v1alpha1/capacity_types.go

```diff
@@ -0,0 +1,35 @@
+/*
+Copyright 2023 Nephio.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import "k8s.io/apimachinery/pkg/api/resource"
+
+func (r *Capacity) GetMaxUplinkThroughput() resource.Quantity {
+	return r.Spec.MaxDownlinkThroughput
```

Comment:
Thinking is there a merit to define an interface for these getters? That may be useful if we just need to provide Get access to the structure underneath which can be done through an interface. This is a general comment applicable for capacity, dnn and interface types in this PR

Reply:
I did something like this in the past and I did not found any benefit doing this. So my view is that it is better to use the method directly on the GoStruct. So an interface adds overhead with little value.